src/Sylius/Component/Originator/Model/OriginAwareTrait.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Originator\Model;
+
+/**
+ * @author Joseph Bielawski <stloyd@gmail.com>
+ */
+trait OriginAwareTrait
```

Comment:
We should remove this trait, I don't think we need it. It's not correct usage of trait, because going in this direction we would have `AdjustmentsAwareTrait`, `ParentAwareTrait`, `NameAwareTrait` and `EverythingAwareTrait`. What is more it's used in only one place.